### ISSUE TITLE
qubes yubikey login stopped working after 6.1.7-1.fc32

### ISSUE THREAD

**Issue by ihateprogramming88** [QubesOS/qubes-issues#8046]
[How to file a helpful issue](https://www.qubes-os.org/doc/issue-tracking/)

### Qubes OS release (if applicable)
xen_version : 4.14.5
linux 6.1.7-1.fc32.qubesx86_64

### Brief summary

After the latest update the yubikey MFA login stopped working. Not sure if this is related to any other problems, however nothing was changed besides the update to latest kernel. 

lightdm nor xscreensaver works

**Comment by marmarek** [comment]
Which approach you use, the one documented at https://www.qubes-os.org/doc/yubikey/#multi-factor-login-for-qubes-os ? If so, dom0 kernel shouldn't matter here, but the one in sys-usb can. Is it also 6.1.7? Which version was there before?

**Comment by ihateprogramming88** [comment]
I did use that guide, the other thing i changed was sys-usb to minimal fedora-37 template, i completely forgot. My apologies. 
What packages do i have to install into the minimal template for it to work again

**Comment by marmarek** [comment]
You have it documented in that guide.

**Comment by andrewdavidwong** [comment]
Closed as "not a bug" (the reported behavior is not a bug and is already working as intended). If anyone believes this is a mistake, please leave a comment, and we'll be happy to take another look. Thank you.

**Comment by UndeadDevel** [comment]
I think this is actually a legitimate bug. Requesting to be re-opened.

I've set up my NitroKey3 with a [modified `qubes-dom0-yubikey`](https://github.com/QubesOS/qubes-app-yubikey/pull/22) and am using it for lock screen and some other 2FA stuff without issue. I cannot get it to work with the initial login screen, however.
Adding it to the `lightdm` PAM module does allow using it when logging in after "Log out" or "Switch user", but not when logging in the first time after boot (even though it's the same login mechanism). I tried adding the `yubikey` module with `include` to PAM modules `login` and `lightdm-autologin` additionally, but to no avail.
